types/ember__test-helpers/index.d.ts

```diff
@@ -223,6 +224,13 @@ declare module '@ember/test-helpers/validate-error-handler' {
     export default function(callback?: (error: Error) => void): { isValid: boolean, message: string };
 }
 
+declare module '@ember/test-helpers/onerror' {
+    import Error from '@ember/error';
+
+    export function setupOnerror(handler: (error: Error) => void): void;
```

Comment:
Actually, thinking on this—is this guaranteed to be an `Error`? I don’t *think* it is?

Reply:
Yeah, you’re right. If you just `throw "foo"` somewhere you’ll get `"foo"` in error handler.
I’ve updated the PR.